cs/forms.texy

```diff
@@ -577,6 +577,13 @@ $form->addText('email', 'E-mail:')->setDisabled(TRUE);
 
 Do tohoto prvku nepůjde zapisovat a jeho hodnota nebude obsažena v datech vracených funkcí `$form->getValues()`.
 
+[.tip]
+Pokud chceme prvek použít jen pro čtení tj. aby se nastavená hodnota prvku zobrazila ale byla deaktivována je potřeba nejdřív prvek deaktivovat a poté mu nastavit hodnotu. Je to z důvodu, že metoda `setDisabled()` vynuluje hodnotu prvku.
```

Comment:
Pokud chceme prvek použít jen pro čtení, tj. aby se nastavená hodnota prvku zobrazila, ale prvek byl neaktivní, je potřeba nejdřív prvek deaktivovat a poté mu nastavit hodnotu. Je to z toho důvodu, že metoda `setDisabled()` hodnotu prvku vynuluje.